Clarification for only using feature IDs (#40)
CesiumGS/glTF: extensions/2.0/Vendor/EXT_mesh_features/README.md

- `featureId.offset` and `featureId.repeat` (for a Feature ID Attribute)
- `featureId.index` (for a Feature ID Texture)

> Feature IDs may be assigned to primitives or nodes 

Reviewer: ```suggestion
Primitives may contain `featureIds` entries for vertex attribute and texture-based feature IDs, and Nodes may contain `featureIds` entries for GPU instance attributes.
```

> #### Resolving Feature IDs with Property Tables

Reviewer: ```suggestion
#### Referencing Property Tables with Feature IDs
```

> }
> ```

#### Referencing External Resources with Feature IDs

> Feature IDs do not have to be associated with a property table. Without a property table, IDs may identify features for use in other extensions or in custom applications. Use cases for these IDs extend beyond the scope of this extension, but could include identifying features for styling or picking, or looking up metadata externally in a REST API or database. These feature IDs can be used by the application in different ways. For example they could be used for styling features in a mesh, or to look up information for these features externally, via a REST API, or in a database.

Reviewer: ```suggestion
Feature IDs do not have to be associated with a property table. Without a property table, IDs may identify features for use in other extensions or in custom applications. Use cases for these IDs extend beyond the scope of this extension, but could include identifying features for styling or picking, or looking up metadata externally in a REST API or database.
```

Ended up with two sentences saying the same thing – i think we could remove the last part, or I'm fine with rephrasing this another way!

Author: This is embarassing. Sorry for not reading the result thoroughly.

(Note to myself: Don't do important commits at 3:00 am....)

> <img src="figures/feature-id-lookup.png"  alt="Feature ID lookup" width="800">

> 

Reviewer: nit: I think my inline suggestion ended up creating extra empty lines here sorry!

> }
> ```

> #### Resolving Feature IDs Externally

Reviewer: ```suggestion
#### Referencing External Resources with Feature IDs
```

> Feature IDs do not have to be associated with a property table. It is also possible to specify feature IDs to only identify the features in an asset, and then use their IDs in an application-specific manner. 

Reviewer: ```suggestion
Feature IDs do not have to be associated with a property table. Without a property table, IDs may identify features for use in other extensions or in custom applications. Use cases for these IDs extend beyond the scope of this extension, but could include identifying features for styling or picking, or looking up metadata externally in a REST API or database.
```

> **Example:** A minimal example for assigning feature IDs to a mesh primitive, using an implicit vertex attribute: The default value for the `offset` is 0, and the value for `repeat` is 3. This means that the vertices will receive the feature IDs `[0, 0, 0, 1, 1, 1, 2, 2, 2, ...]`, causing the vertices of each triangle to receive the same feature ID. 
>
> ```jsonc
> // Primitive:
> "extensions": {
>   "EXT_mesh_features": {
>     "featureIds": [
>       { "repeat": 3 }
>     ]
>   }
> }

> > ```

Reviewer: With the illustration below (thanks!), I think it'd be fine to omit the example JSON here. I was going to suggest simplifying to `{"attribute": 0}` but it doesn't really add new information that we haven't seen earlier.

> These feature IDs can be used by the application in different ways. For example they could be used for styling features in a mesh, or to look up information for these features externally, via a REST API, or in a database.

Reviewer: ```suggestion
```

^Suggest merging this into the previous paragraph; see comment above.

Author: Indeed, the example did not add sooo much value here (and did not really "match" the diagram in any way, FWIW). 
I have removed it, and applied the other changes that you suggested.
